### DISCUSSION TITLE
Update news?

### DISCUSSION THREAD

**Discussion by E2GO** [lllyasviel/Fooocus#2304] [General]
Hey! Any news on updates for Fooocus? 70+ pull requests future?

**Comment by barepixels** [comment]
The boss is probably busy with Stable Difussion Forge

**Reply by mashb1t** [reply]
yes, this. I'm working selected PRs (now all grouped by size) and am going to merge + already have started merging some of them to develop, but my time is limited. Trying my best, last release already included 15+ bugfixes, see https://github.com/lllyasviel/Fooocus/releases/tag/2.1.865.

**Reply by barepixels** [reply]
thank you

**Reply by E2GO** [reply]
TY for info!

**Reply by f0n51** [reply]
Sad to see this very promising project is stagnating. While I can fully understand lllyasviel focusing on other projects and all other contributors doing this in their free time, is there any way to push Fooocus forward?

**Reply by mashb1t** [reply]
https://github.com/lllyasviel/Fooocus/discussions/2362

You can also check if sdxl lightning can be a replacement of LCM or a 4th performance option

**Comment by rionix88** [comment]
will foocus add support for pose control net?
